src/bevy_setup/mod.rs

```diff
@@ -0,0 +1,32 @@
+use bevy::asset::AssetMetaCheck;
```

Comment:
Any reason this file is `bevy_setup/mod.rs` instead of `bevy_setup.rs`?

Reply:
Consistency with the others. But you're right, I'll change it.